TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -186,7 +186,7 @@ public FileCompiler OpenTextDocument(DocumentContext docContext, string sourceTe
             }
 
             var matchingPgm =
-                _customSymbols.Programs.Keys.FirstOrDefault(
+                _customSymbols?.Programs.Keys.FirstOrDefault(
                     k => k.Equals(inputFileName, StringComparison.InvariantCultureIgnoreCase));
             if (matchingPgm != null)
             {
```

Comment:
For performance reason, please use StringComparison.OrdinalIgnoreCase

Reply:
See 3ad742f7cc454fc77569047c70804b8f20f1c9de.